pytradfri/api/aiocoap_api.py

```diff
@@ -130,6 +152,8 @@ def error_callback(ex):
         ob = pr.observation
         ob.register_callback(success_callback)
         ob.register_errback(error_callback)
+        nonlocal _observations_err_callbacks
```

Comment:
Me neither, but I see `UnboundLocalError: local variable '_protocol' referenced before assignment` if it's not there. How does Python evaluate that?